### DISCUSSION TITLE
can not install apk (begginer question)

### DISCUSSION THREAD

**Discussion by leostereo** [quasarframework/quasar#8870] [CLI - Capacitor / Cordova modes]
Hi guys , I was trying to build my first app with cordova following official documentation , on recently created project did:
```
npm install -g cordova
quasar create mobile_test && cd mobile_test
quasar mode add cordova && cd src-cordova
cordova platform add android
cd src-cordova/
cordova platform add android
quasar dev -m cordova -T android
cd ..
quasar dev -m cordova -T android
(now quasar default app is properly running on my virtual device... this is good!)
quasar dev --mode cordova --target android
(now I have an instalable apk ... and it works in my android device ... this is good!!)
quasar build --mode cordova --target android
```
After try installing the "app-release-unsigned.apk" , I got application not installed.
Why is that ? what is missing ? what is the difference between the dev and build generated apks  ?
Thanks ...
btw ... quasar is awesome.

**Comment by metalsadman** [comment]
If you are just testing, then you can add a `--debug` flag on your build command ie. `quasar build -m cordova --debug`, otherwise see [publishing to store](https://quasar.dev/quasar-cli/developing-cordova-apps/publishing-to-store).

**Reply by leostereo** [reply]
Thanks!!! didnt see that at docs.
